pkg/operator/starter.go

```diff
@@ -572,7 +572,7 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		operatorCtx.operatorClient.Informers,
 		eventRecorder)
 
-	auditPolicyPahGetter, err := libgoassets.NewAuditPolicyPathGetter()
+	auditPolicyPahGetter, err := libgoassets.NewAuditPolicyPathGetter("/var/run/configmaps/audit")
```

Comment:
nit: not coming from your changes but maybe a good time to fix the variable name to `auditPolicyPathGetter`?